pkg/gardenlet/apis/config/helper/helpers.go

```diff
@@ -109,3 +109,12 @@ func IsLokiEnabled(c *config.GardenletConfiguration) bool {
 	}
 	return true
 }
+
+// IsMonitoringEnabled return true if the monitoring stack for clusters is enabled. Default is enabled.
+func IsMonitoringEnabled(c *config.GardenletConfiguration) bool {
```

Comment:
Can you add a unit test for this function?

Reply:
done!